playground/define/vite.config.js

```diff
@@ -29,5 +29,6 @@ export default defineConfig({
     ÖUNICODE_LETTERɵ: 789,
     __VAR_NAME__: false,
     __STRINGIFIED_OBJ__: JSON.stringify({ foo: true }),
+    'import.meta.env.LEGACY': '__VITE_IS_LEGACY__',
```

Comment:
I think we should probably define this like `'import.meta.env.VAR': '__VITE_VAR__'` or something else, to prevent confusion/conflicts with how `plugin-legacy` already works.